packages/block-library/src/media-text/style.scss

```diff
@@ -1,7 +1,9 @@
-.wp-block-media-text {
+.wp-gs .wp-block-media-text {
```

Comment:
mmm, we need to remove this style block as well, right?

Reply:
missed it